Gems/Profiler/Code/Source/CpuProfiler.cpp

```diff
@@ -406,6 +407,7 @@ namespace Profiler
                 }
             }
         }
+        m_timeTicksPerSecond = AZStd ::GetTimeTicksPerSecond();
```

Comment:
This should use [AZStd::chrono::steady_clock::now()](https://en.cppreference.com/w/cpp/chrono/steady_clock) to retrieve a monotonic clock value that isn't affected by the system clock.

Reply:
Because TimeTicksPeSecond is hard-coded value based on a certain frequency of time.
On Linux and Android this is just returns [1'000'000'000](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/Platform/Common/UnixLike/AzCore/std/time_UnixLike.cpp#L17-L19).
On Windows it is value of [QueryPerformanceFrequency](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/Platform/Windows/AzCore/std/time_Windows.cpp#L19)

There is no reason to store the value or even serialize the value out

Reply:
Well if the profiler works in ticks, then the existing data is already not portable.

But that does seem like a larger refactor of the entire, so I'll drop this request.